test/roby_app/test_plugin.rb

```diff
@@ -212,6 +212,143 @@ def perform_app_assertion(result)
                     perform_app_assertion interface.unit_tests.orogen_deployment_exists?
                 end
             end
+
+            describe "Local Log Transfer Server" do
+                def pre_config_server
+                    @app = Roby::Application.new
+                    @app.log_dir = make_tmpdir
+                    @current_log_level = RemoteProcesses::Server.logger.level
+                    RemoteProcesses::Server.logger.level = Logger::FATAL + 1
+                end
+
+                def start_server
+                    raise "server already started" if @server
+
+                    @server = RemoteProcesses::Server.new(@app, port: 0)
+                    server.open
+                    @server_thread = Thread.new { server.listen }
+                end
+
+                def connect_to_server
+                    @root_loader = OroGen::Loaders::Aggregate.new
+                    OroGen::Loaders::RTT.setup_loader(root_loader)
+                    @client = RemoteProcesses::Client.new(
+                        "localhost",
+                        server.port,
+                        root_loader: root_loader
+                    )
+                end
+
+                def start_and_connect_to_server
+                    start_server
+                    connect_to_server
+                end
+
+                def start_process_server(name, host)
```

Comment:
There are too many servers in these method names. I think this would be best named "register_process_server"